dev-kubeflow-org/recreate_app.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+#
+# A simple script to recreate the Kubeflow app for kubeflow-dev.
+# The purpose of this script is to make it easy to pull in the latest
+# release.
+#
+# By design this script doesn't actually modify the current deployment
+# (e.g. delete the current namespace or apply the deployment)
+set -ex
+# Create a namespace for kubeflow deployment
+NAMESPACE=kubeflow
+
+# Which version of Kubeflow to use
+# For a list of releases refer to:
+# https://github.com/kubeflow/kubeflow/releases
+VERSION=v0.1.0-rc.2
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+cd ${DIR}
+
+APP_NAME=ks-app
+
+
+if [ -d ${DIR}/${APP_NAME} ]; then
+	# TODO(jlewi): Maybe we should prompt to ask if we want to delete?
+	echo "Directory ${DIR}/${APP_NAME} exists; please delete it first"
+	exit 1
+fi
+
+# Initialize a ksonnet app. Set the namespace for it's default environment.
+
+ks init ${APP_NAME}
+cd ${APP_NAME}
+ks env set default --namespace ${NAMESPACE}
+
+# Install Kubeflow components
+ks registry add kubeflow github.com/kubeflow/kubeflow/tree/${VERSION}/kubeflow
+
+ks pkg install kubeflow/core@${VERSION}
+ks pkg install kubeflow/tf-serving@${VERSION}
+ks pkg install kubeflow/tf-job@${VERSION}
+
+# Create templates for core components
+ks generate kubeflow-core kubeflow-core
+
+# Setup ingress
+ACCOUNT=google-kubeflow-team@google.com
+CLIENT_ID="235037502967-9cpmvs4ljbiqb3ojtnhnhlkkd8d562rl.apps.googleusercontent.com"
```

Comment:
@danisla is about to merge https://github.com/kubeflow/kubeflow/pull/572/ which migrates CLIENT_ID and CLIENT_SECRET to k8s secrets and removes these ksonnet params. 

Can you update this script to use
```
kubectl -n ${NAMESPACE} create secret generic kubeflow-oauth --from-literal=CLIENT_ID=${CLIENT_ID} --from-literal=CLIENT_SECRET=${CLIENT_SECRET}
```